### PR TITLE
Improved heart in footer look

### PR DESCRIPTION
**Pull request by nelsonmestevao**
I think it looks nicer in all platforms.

### CODE REVIEW

File: app/views/layouts/lp/_footer.html.slim [hunk 1]
```diff
@@ -50,4 +50,4 @@ footer#footer
 
   .footer-btm
     .container.text-center
-      span made with &hearts; by CeSIUM
+      span made with <i class="fa fa-heart" style="color: #ED4337;" aria-hidden="true"></i> by CeSIUM
```
Comment (marcelo140) [inline app/views/layouts/lp/_footer.html.slim]: If you want this to be merged you will have to change it to slim yourself. Check the documentation for help: http://www.rubydoc.info/gems/slim/frames

Furthermore, HTML and CSS should not be mixed. If you need to add/change css use the stylesheets: `app/assets/stylesheets`.

**Comment by nelsonmestevao** [comment]
I think I got it right now.

**Comment by nelsonmestevao** [comment]
I changed the color of the heart to a variable called `$soft-red`.

I wrote the non-breaking space character `&nbsp;` as a way to force a space between the heart.

File: app/views/layouts/lp/_footer.html.slim [hunk 2]
```diff
@@ -50,4 +50,7 @@ footer#footer
 
   .footer-btm
     .container.text-center
-      span made with &hearts; by CeSIUM
+      span
+       | made with &nbsp;
```
Comment (marcelo140) [inline app/views/layouts/lp/_footer.html.slim]: To add a trailing whitespace you should use the [' operator](http://www.rubydoc.info/gems/slim/frames#Verbatim_text_with_trailing_white_space__).

File: app/views/layouts/lp/_footer.html.slim [hunk 3]
```diff
@@ -50,4 +50,7 @@ footer#footer
...
+       i.fa.fa-heart 
+       | &nbsp; by CeSIUM
```
Comment (marcelo140) [inline app/views/layouts/lp/_footer.html.slim]: Here you can simply add an extra whitespace as documented in [slim](http://www.rubydoc.info/gems/slim/frames#Verbatim_text__)

Comment (marcelo140) [inline app/views/layouts/lp/_footer.html.slim]: Here you can simply add an extra whitespace as documented in [slim](Here you can simply add an extra whitespace as documented in slim).

Comment (nelsonmestevao) [reply app/views/layouts/lp/_footer.html.slim]: should I keep the `|`?

File: app/assets/stylesheets/variables.scss [hunk 4]
```diff
@@ -10,3 +10,4 @@ $color-feature: #EB7955;            // feature color
 $base-line-height: 24px;
 $primary-font: 'Lato', sans-serif;  // default font family
 $minimum-font-size: 13px
```
Comment (martinhoaragao) [inline app/assets/stylesheets/variables.scss]: ";" lacking, causes a syntax error.

**Pull request merged**